opentelemetry-user-events-logs/Cargo.toml

```diff
@@ -26,5 +26,5 @@ tracing-subscriber = { version = "0.3.0", default-features = false, features = [
 microbench = "0.5"
 
 [features]
-logs_level_enabled = ["opentelemetry/logs_level_enabled", "opentelemetry_sdk/logs_level_enabled"]
+logs_level_enabled = ["opentelemetry/spec_unstable_logs_enabled", "opentelemetry_sdk/spec_unstable_logs_enabled", "opentelemetry-appender-tracing/spec_unstable_logs_enabled"]
```

Comment:
should the feature here also be renamed to reflect the unstable nature?

Reply:
Not necessarily, unless we want to be consistent with same name across all the crates.

Reply:
But better to be consistent across. Have renamed it now :)